app/assets/javascripts/fae/form/_validator.js

```diff
@@ -84,9 +88,11 @@ Fae.form.validator = {
 
           $this.submit();
         } else {
-          // otherwise scroll to the top to display alerts
+          // otherwise scroll to the top to display alerts (unless in a nested form scope)
           Fae.navigation.language.checkForHiddenErrors();
-          FCH.smoothScroll($('#js-main-header'), 500, 100, 0);
+          if (typeof($scope) === 'undefined') {
```

Comment:
This prevents the page from scrolling all the way back to the top of the parent form when you clear out a selected item in a nested form at the bottom of the page.